spec/person_spec.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'spec_helper'
+
+describe Person do
+  person = Person.new(25, 25, name: 'jex')
+  it 'should be an instance of Person' do
+    expect(person).to be_an_instance_of(Person)
+  end
+  it 'should have a name' do
+    expect(person.name).to eql('jex')
+  end
+  it 'should have an age' do
+    expect(person.age).to eql(25)
+  end
+  it 'should have an id' do
+    expect(person.id).to be_an_instance_of(Integer)
+  end
+end
```

Comment:
- [ ] Great job so far 👏🏻 👏🏻 However, according to the project requirements, there should be unit tests for all the methods in all the classes, Kindly make sure the `of_age?, can_use_services?, correct_name and add_rental` class person method has (at least) an 'it' block 👇🏻 check your person.rb file

```ruby
  def can_use_services?
    of_age? || @parent_permission
  end

 def of_age?
    @age > 18
  end

  def correct_name
    @name
  end

  def add_rental(person, date)
    @rentals.push(person, date, self)
  end
```